internal/utility/fileposition/string.go

```diff
@@ -55,14 +55,14 @@ func QuoteMetaDelimiters(prefix string, suffix string) (string, string) {
 }
 
 func ExtractDelimitedRegexpPositionInBlock(block []string, str string, blockStartLine int, prefix string, suffix string) *models.FilePosition {
-	group := fmt.Sprintf("(%s)", str)
+	term := "(?P<term>" + str + ")"
```

Comment:
Can you add a function comment explaining what it does and how the regex works please ?

Reply:
Comment included + Unit test added for the prefix/suffix with group case